### ISSUE TITLE
Combining Glyph Tables are Inaccurate

### ISSUE THREAD

**Issue by tobyrush** [fontforge/fontforge#3306]
The FontForge tables used to determine components for accented characters and ligatures looks like it has errors that aren't reflected in the Unicode tables.

For example, FontForge reports that U+0122, LATIN CAPITAL LETTER G WITH CEDILLA, is (according to Elements > Glyph Info > Components) comprised of U+0047 and U+0326, but [http://unicode.org/Public/UNIDATA/UnicodeData.txt](http://unicode.org/Public/UNIDATA/UnicodeData.txt) correctly indicates it should be U+0047 and U+0327. I've found at least two other similar errors in that same Unicode neighborhood.

I don't trust myself to dive into the code but wanted to report the problem. I'm using the 2017-07-30 build on a Mac OS X 10.13.5 beta (17F70a, though I don't imagine it matters much) in XQuartz 2.7.11. I just downloaded and tried it in the daily build FontForge-2018-06-02-9d42597 and the problem is still present there.

**Comment by JoesCat** [comment]
Hi @tobyrush,
At the moment, nothing has changed from 2017-07-30 yet in terms of the problem.
There are a couple of places where some decisions are hardcoded into the FontForge code independent of what the Unicode tables show.

In terms of ligatures, the code in Unicode/is_Ligature*.* extracts the code from the unicode tables and is currently working okay for native scripting and python scripting, but has not been added yet to the FontForge code itself. Doing a grep of some random codes, ex 0132 shows:
```sh
grep "0132" -R *
fontforgeexe/charinfo.c:      case 0x0132: case 0x0133:          /* ij, IJ */
```
From the code listed here, charinfo.c is a point of interest that may be something you may want to look at, to see if the hardcoded ligatures can make use of function calls in Unicode/is_Ligature.c

Accented characters look like another candidate for something similar to is_Ligature, but no code is written for it yet, so we're still dependant on the hardcoded checks ATM.

Hopefully this helps if you still want to dive-in and give it a try (for ligatures).

**Comment by tobyrush** [comment]
For the specific issue I mentioned above, I was able to track it down to fvcomposite.c around line 1199, where there are some hardcoded exceptions switching cedillas (cedillae?) into commas. Further research led me to this [interesting typographic tidbit](https://en.wikipedia.org/wiki/Cedilla#Latvian). Turns out it's Unicode, not FontForge that is wrong!

So I now feel completely justified in not trusting myself to muck with the code. Thanks for the help!
